src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ItemsPresenter.cs

```diff
@@ -0,0 +1,1004 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.UI;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Markup;
+using Windows.UI.Xaml.Media;
+using FluentAssertions;
+using MUXControlsTestApp.Utilities;
+using static Private.Infrastructure.TestServices;
+using Uno.UI.RuntimeTests.Extensions;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+using Uno.Extensions;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls;
+
+public static class AssertionExtensions
+{
+	public static AndConstraint<FluentAssertions.Collections.GenericCollectionAssertions<float>> BeEquivalentToWithTolerance<T>(this FluentAssertions.Collections.GenericCollectionAssertions<float> assertions, float[] expected, float tolerance)
+	{
+		assertions.Subject.Should().HaveCount(expected.Length);
+
+		var subject = assertions.Subject.ToArray();
+
+		for (var i = 0; i < subject.Length; i++)
+		{
+			var sub = subject[i];
+			var actual = expected[i];
+			sub.Should().BeApproximately(actual, tolerance);
+		}
+
+		return new AndConstraint<FluentAssertions.Collections.GenericCollectionAssertions<float>>(assertions);
+	}
+}
+
+[TestClass]
+public class Given_ItemsPresenter
+{
+	// Due to physical/logical pixel conversion on Android, measurements aren't exact
+	private float Epsilon =>
+#if XAMARIN
+			0.5f;
+#else
+		0.0f;
+#endif
+
+
+	[TestMethod]
+	[RunsOnUIThread]
+	[RequiresFullWindow]
```

Comment:
Is it really required for the test to be full screen?

Reply:
Full window is not strictly necessary, but I'm trying to get as much width and height as I can. I've already had to reduce the width and height of the tests multiple times (different platforms have different dimensions + local vs CI).